pkg/storage/tpr/storage_interface.go

```diff
@@ -106,8 +106,18 @@ func (t *store) Create(
 		t.singularKind.URLName(),
 	).Body(data)
 
+	res := req.Do()
+	if res.Error() != nil {
+		glog.Errorf("executing DELETE for %s/%s (%s)", ns, name, res.Error())
+	}
+	var statusCode int
+	res.StatusCode(&statusCode)
+	if statusCode == http.StatusConflict {
+		return storage.NewKeyExistsError(key, 0)
+	}
```

Comment:
maybe we should bail with a generic error if there's any other `err != nil` or unexpected status code